### PR TITLE
Test for the Driver model #148

### PR DESCRIPTION
**Pull request by dcarranza9**
Test for the Driver model #148 done!

### CODE REVIEW

File: sources/tests/test_models.py [hunk 1]
```diff
@@ -31,3 +31,35 @@ def test_delete_parcel(self):
         self.parcel.delete()
         self.assertNotIn(self.parcel, self.queryset)
 
+
+@tag('driver')
+class TestParcel(TestCase):
```
Comment (danialeresbar) [inline sources/tests/test_models.py:36]: Maybe TestDriver?

**Review by danialeresbar — COMMENTED** [review]
Please check my suggestions

File: sources/tests/test_models.py (hunk 1 above)
Comment (dcarranza9) [reply sources/tests/test_models.py]: Yes. adjustment done. Please check newest commit. 

Thank you!

**Comment by jmencisom** [comment]
When you have the chance, @danialeresbar please let us know if this PR is going to be approved. Thanks.

**Review by danialeresbar — APPROVED** [review]
It is normal that as developers we have our own style of testing things but, it would be great if for future similar topics you could use a little more realistic data.
Thanks 😁

**Pull request merged**